samples/zoom-time.html

```diff
@@ -127,7 +131,7 @@
 		};
 
 		window.toggleDragMode = function() {
-			var zoomOptions = window.myLine.options.zoom;
+			var zoomOptions = window.myLine.options.zoom || window.myLine.options.plugins.zoom.zoom;
```

Comment:
This can just be:

    var zoomOptions = window.myLine.options.plugins.zoom.zoom

Reply:
ok, changed.